cmd/deploy/deploy.go

```diff
@@ -408,3 +416,199 @@ func (d *deployCommand) getResourceStreamClient(
 	}
 	return stream, nil
 }
+
+func (d *deployCommand) processJobDeploymentResponses(namespaceName string, stream pb.JobSpecificationService_DeployJobSpecificationClient) (string, error) {
```

Comment:
Processing Job Deployment responses should be same in the refresh command too lets just reuses the same.